scapy/layers/tls/handshake.py

```diff
@@ -1179,7 +1179,7 @@ class TLSNewSessionTicket(_TLSHandshake):
     @classmethod
     def dispatch_hook(cls, _pkt=None, *args, **kargs):
         s = kargs.get("tls_session", None)
-        if s and s.tls_version >= 0x0304:
+        if s and s.tls_version and s.tls_version >= 0x0304:
```

Comment:
Did you find a bug with this line ? It will be nice to add a test.
I guess that it would be better to test that s is an instance of `tlsSession`

Reply:
Test added !

Reply:
Can `s.tls_version` be None ? If the answer is yes, could you add a test to check for this specific corner case ?

Reply:
Already included in https://github.com/secdev/scapy/pull/992/files#diff-cd90a85f0ec6440949616e4896487acaR14

Reply:
Sorry, I meant rewrite line 1182 as:
`if s and isinstance(s, tlsSession) and s.tls_version >= 0x0304:`
and make sure that the unit test sets the "tls_session" kwargs as None to check that this new test works.

Reply:
Ok i've added an explicit test :)

Reply:
Sorry I had not understood what you meant, I added the `isinstance` test